react/components/molecules/key-values-placeholder/key-values-placeholder.js

```diff
@@ -0,0 +1,36 @@
+import React, { PureComponent } from "react";
+import { View, ViewPropTypes } from "react-native";
+import PropTypes from "prop-types";
+
+import { KeyValuePlaceholder } from "../key-value-placeholder";
+
+export class KeyValuesPlaceholder extends PureComponent {
+    static get propTypes() {
+        return {
+            colors: PropTypes.arrayOf(PropTypes.string).isRequired,
+            locations: PropTypes.arrayOf(PropTypes.number).isRequired,
+            numberOfItems: PropTypes.number,
+            style: ViewPropTypes.style
+        };
+    }
+
+    static get defaultProps() {
+        return { numberOfItems: 1, style: {} };
+    }
+
+    render() {
+        return (
+            <View style={this.props.style}>
+                {[...Array(this.props.numberOfItems)].map((_item, index) => (

```

Comment:
Can't `Array(this.props.numberOfItems)` be used instead of `[...Array(this.props.numberOfItems)]`?

Reply:
It wouldn't work.
The initialisation of a new array only sets the length. For maps to work it requires to something to exist in the positions even if it is undefined. 
<img width="1091" alt="Screenshot 2020-04-14 at 14 14 27" src="https://user-images.githubusercontent.com/13239857/79229709-65026180-7e5b-11ea-820e-9ce3932d6770.png">